input/cql/IMMZCommon.cql

```diff
@@ -424,6 +424,30 @@ define "Typhoid Doses Administered to Patient":
   where
     I.vaccineCode in IMMZc."Typhoid Vaccine"
 
+/** 
+ * @dataElement No Typhoid doses administered to patient
+ */
+define "No Typhoid Doses Administered to Patient":
+  not exists ("Typhoid Doses Administered to Patient")
+
+/** 
+ * @dataElement Largest sequence of typhoid doses administered to patient
+ */
+define "Highest Sequence of Typhoid Doses Administered to Patient":
```

Comment:
@goelalex / @mo3dev / @nityan  - This uses the highest sequence over the count - this is because Typhoid can have restarts of the schedule (so it might be 1, 2, 1, 2, 3 which is 3rd dose delivered via 5 injections). I think that the other antigens can also use this logic - did we want to leave the other DTs as "number of X" or did we want to swap them over to be more consistent? (I'm not sure if a `Count()` performs any different than a `Max()`)